test/unit/streaming.rules.abr.SwitchHistoryRule.js

```diff
@@ -0,0 +1,22 @@
+import SwitchHistoryRule from '../../src/streaming/rules/abr/SwitchHistoryRule';
+
+import RulesContextMock from './mocks/RulesContextMock';
+
+const expect = require('chai').expect;
+const context = {};
+const switchHistoryRule = SwitchHistoryRule(context).create();
+
+describe('SwitchHistoryRule', function () {
+    it("should return an empty switchRequest when getMaxIndex function is called with an empty parameter", function () {
+        const switchRequest = switchHistoryRule.getMaxIndex();
+
+        expect(switchRequest.quality).to.be.equal(-1);  // jshint ignore:line 
```

Comment:
@nicosang, thank you very much. PR looks great. Just one minor proposal. Could we use SwitchRequest.NO_CHANGE here instead of -1?